src/routes/results/[results_string]/+page.ts

```diff
@@ -0,0 +1,19 @@
+import { error } from '@sveltejs/kit';
+
+/** @type {import('./$types').PageLoad} */
+export function load({ params }: { params: { results_string: string } }) {
+	const matches = [...params.results_string.matchAll(new RegExp(/([a-h])([1-5])/gi))];
```

Comment:
Cool! :) I was wondering what your approach would be for the mapping. An alternative could be doing something like a base64 encoding of a stringified json object, but as long as the order of the dynamics aren't expected to change I love the simplicity and conciseness of this! Maybe we can later add to the README an extra note about this mapping for future devs.